### PR TITLE
Fixes Chainlink Fences Being Leapable

### PR DESCRIPTION
**Pull request by RustingWithYou**
Mobs can no longer leap over the Konyang chainlink fences.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!bugfix

**Comment by RustingWithYou** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]

File: code/game/objects/structures/urban.dm [hunk 1]
```diff
@@ -351,16 +351,21 @@
...
+	if(istype(mover, /obj/item/projectile))
+		var/obj/item/projectile/P = mover
+		if(P.original == src)
+			return FALSE
+		if(P.firer && Adjacent(P.firer))
+			return TRUE
+		return prob(35)
+	if(isliving(mover))
+		return FALSE
+	if(istype(mover) && mover.checkpass(PASSTABLE))
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/urban.dm:366]: Passtable is probably a bit goofy on a tall chainlink fence

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/urban.dm (hunk 1 above)
Comment (RustingWithYou) [reply code/game/objects/structures/urban.dm:366]: This was taken from the other chainlink fence in code which can't be leapt over. If there's a better flag to use here LMK

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**